### PR TITLE
Add an IntelliJ runConfig for the Gallery example

### LINKED ISSUE #6666: Gallery missing "run configs" for IntelliJ
I'd like to open the Gallery in IntelliJ and then run it, via IntelliJ.

It looks like we don't have run configs checked in for the gallery, so when I open/import Gallery into IntelliJ, I see this:

screen shot 2016-11-02 at 2 19 47 pm

Notice how I don't have any run configs here.

Should we check in configs for flutter's default modes? (debug, profile, release)

### PR DESCRIPTION
**Pull request by mit-mit**
Fixes #6666

### CODE REVIEW

**Comment by sethladd** [comment]
travis is sad, not sure why?

**Comment by mit-mit** [comment]
It's failing many PRs back from this one...

File: examples/flutter_gallery/.gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 .DS_Store
 .atom/
-.idea
```
Comment (devoncarew) [inline examples/flutter_gallery/.gitignore]: You'll probably want to keep this exclusion. You can still commit individual files in `.dea/`, you just need to use `-f` when adding the file.

**Comment by devoncarew** [comment]
lgtm modulo the .gitignore change

**Comment by mit-mit** [comment]
Thanks Devon, added back .idea to .gitignore

**Comment by mit-mit** [comment]
Travis passes after merging with master. Submitting.

**Comment by tvolkert** [comment]
I doesn't look like the .gitignore change was included in the merge.

**Comment by devoncarew** [comment]
Yup, we didn't want that change, just the addition of the run configuration (there are still several files in .idea that we want to ignore).

**Pull request merged**